db/migrations/20160908142125_ewycd-2-definition.js

```diff
@@ -1,56 +1,61 @@
 /* eslint-disable quote-props, max-len */
```

Comment:
there is no amendments here. just refactoring internal structure to support tests.